.github/workflows/integration_tests.yml

```diff
@@ -879,3 +879,166 @@ jobs:
+        uses: dawidd6/action-download-artifact@v2.14.0
+        with:
+          # Download last successful artifact from a CI build
+          github_token: ${{secrets.GITHUB_TOKEN}}
+          workflow: CI.yml
+          #event: push
+          workflow_conclusion: success
+          # use the following branch
+          branch: ${{ env.BRANCH}}
+          # directory where to extract artifacts to
+          path: ./dist
+
+      - name: Install Keptn CLI
+        id: install_keptn_cli
+        run: |
+          mkdir ${HOME}/downloads
+          wget "https://github.com/keptn/keptn/releases/download/${{ env.META_KEPTN_VERSION }}/keptn-${{ env.META_KEPTN_VERSION }}-linux-amd64.tar.gz" -O "${HOME}/downloads/keptn-cli.tar.gz"
+          tar -zxvf ${HOME}/downloads/keptn-cli.tar.gz
+          sudo mv keptn-*-linux-amd64 /usr/local/bin/keptn
+
+      - name: Deploy Keptn with Keptn
+        id: deploy_keptn_with_keptn
+        run: |
+          KEPTN_HELM_CHART_FILENAME=$(ls dist/keptn-installer/keptn*.tgz | head -1)
+          HELM_SERVICE_HELM_CHART_FILENAME=$(ls dist/keptn-installer/helm-service*.tgz | head -1)
+          JMETER_SERVICE_HELM_CHART_FILENAME=$(ls dist/keptn-installer/jmeter-service*.tgz | head -1)
+
+          keptn auth --endpoint=${{ secrets.META_KEPTN_API_URL }} --api-token=${{ secrets.META_KEPTN_API_TOKEN }}
+          keptn add-resource --project=${{ env.META_KEPTN_KEPTN_PROJECT }} --service=keptn --all-stages --resource=${KEPTN_HELM_CHART_FILENAME} --resourceUri=helm/keptn.tgz
+          keptn add-resource --project=${{ env.META_KEPTN_KEPTN_PROJECT }} --service=helm-service --all-stages --resource=${HELM_SERVICE_HELM_CHART_FILENAME} --resourceUri=helm/helm-service.tgz
+          keptn add-resource --project=${{ env.META_KEPTN_KEPTN_PROJECT }} --service=jmeter-service --all-stages --resource=${JMETER_SERVICE_HELM_CHART_FILENAME} --resourceUri=helm/jmeter-service.tgz
+
+      - name: Trigger Keptn Control Plane Delivery
+        id: trigger-keptn-control-plane-delivery
+        uses: keptn/gh-action-send-event@main
+        with:
+          keptnApiUrl: ${{ secrets.META_KEPTN_API_URL }}/v1/event
+          keptnApiToken: ${{ secrets.META_KEPTN_API_TOKEN }} # TODO: add github action run as label
+          event: |
+            {
+              "data": {
+                "configurationChange": {
+                  "values": {
+                    "control-plane": {
+                      "enabled": true
+                    }
+                  }
+                },
+                "deployment": {
+                  "deploymentURIsLocal": ["http://api-gateway-nginx.keptn-hardening"],
+                  "deploymentstrategy": "direct"
+                },
+                "message": "",
+                "project": "${{ env.META_KEPTN_KEPTN_PROJECT }}",
+                "result": "",
+                "service": "keptn",
+                "stage": "dev",
```

Comment:
We use `"stage": "dev"` here, but 
```
                   "deploymentURIsLocal": ["http://api-gateway-nginx.keptn-hardening"],
```
suggests there also is a hardening stage. Which one is correct?